File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = Integer.MIN_VALUE;
        int Onescount = 0;
        for(int ele : nums){
            if(ele == 1){
                Onescount++;
            }
            else if(ele == 0){
                Onescount = 0;
            }
             max = Math.max(max,Onescount);
        }
        return max;
    }
}

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxOnes = 0
        count = 0
        for i in range(len(nums)):
            if(nums[i] == 1):
                count += 1
                maxOnes = max(maxOnes,count)
            elif(nums[i] == 0):
                count = 0
        return maxOnes
        